node-feature-discovery-0.15.yaml

```diff
@@ -26,6 +26,31 @@ pipeline:
       mkdir -p ${{targets.contextdir}}/etc/kubernetes/node-feature-discovery/
       cp ./deployment/components/worker-config/nfd-worker.conf.example ${{targets.contextdir}}/etc/kubernetes/node-feature-discovery/nfd-worker.conf
 
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0
+      modroot: .
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0
+      modroot: .
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0
+      modroot: .
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.23.0
+      modroot: .
+
+  - uses: go/bump
+    with:
```

Comment:
looks like the bot will create multiple bumps, one for each go build pipeline below but we don't include the correct dir for the `modroot`.  That said, not sure if x  net is in each of these.